modules/openapi-generator/src/main/resources/kotlin-server/libraries/ktor/Paths.kt.mustache

```diff
@@ -18,7 +18,7 @@ object Paths {
      {{#allParams}}* @param {{paramName}} {{description}} {{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
      {{/allParams}}*/
     @KtorExperimentalLocationsAPI
-    @Location("{{path}}") class {{operationId}}({{#allParams}}val {{paramName}}: {{{dataType}}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
+    @Location("{{path}}") class {{operationId}}({{#allParams}}val {{paramName}}: {{{dataType}}}{{^required}}? = null{{/required}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
```

Comment:
Maybe we should also check for {{#isNullable}}?{{/isNullable}}

Reply:
Thank you for reviewing!!

How is this? -> 

```suggestion
    @Location("{{path}}") class {{operationId}}({{#allParams}}val {{paramName}}: {{{dataType}}}{{#isNullable}}?{{/isNullable}}{{^isNullable}}{{^required}}? = null{{/required}}{{/isNullable}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
```


I couldn't test because I don't know how to send `null` in query parameters.

Reply:
What about something like this?

`@Location("{{path}}") class {{operationId}}({{#allParams}}val {{paramName}}: {{{dataType}}}{{#isNullable}}?{{^required}} = null{{/required}}{{/isNullable}}{{^isNullable}}{{^required}}? = null{{/required}}{{/isNullable}}{{#hasMore}}, {{/hasMore}}{{/allParams}})`